apps/login/lib/demos.ts

```diff
@@ -48,11 +48,11 @@ export const demos: { name: string; items: Item[] }[] = [
       //     slug: "passwordless",
       //     description: "The page to login a user with his passwordless device",
       //   },
-      //   {
-      //     name: "Passwordless Create",
-      //     slug: "passwordless/create",
-      //     description: "The page to add a users passwordless device",
-      //   },
+      {
```

Comment:
Set Password is also available, right?
BTW, what's the difference between set and change pw?

Reply:
> but the idea is to have an endpoint where I can change my password with an active session

And what does the other endpoint? Just for my understanding. I also didn't find the answer in the epics issues.

Reply:
which one? password?

Reply:
is "reset" mean with "set", so the user has to change it?